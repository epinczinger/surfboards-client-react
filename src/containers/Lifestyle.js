import React from 'react';

const Lifestyle = () => (
  <div className="w-screen p-4 shadow">
    <iframe
      src="https://www.youtube.com/embed/gpPY-mqGXKs"
      title="Youtube video"
      frameBorder="0"
      allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
      allowFullScreen
      width="100%"
      height="100%"
    />
  </div>
);

export default Lifestyle;
