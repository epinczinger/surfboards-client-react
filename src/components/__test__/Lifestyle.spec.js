import React from "react";
import renderer from "react-test-renderer";
import Lifestyle from "../Lifestyle";

describe("The Lifestyle component", () => {
  it("renders as expected matching snapshot", () => {
    const tree = renderer.create(<Lifestyle />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
