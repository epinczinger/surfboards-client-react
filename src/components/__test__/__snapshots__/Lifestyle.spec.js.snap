// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Lifestyle component renders as expected matching snapshot 1`] = `
<div
  className="w-screen p-4 shadow"
>
  <iframe
    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
    allowFullScreen={true}
    frameBorder="0"
    height="100%"
    src="https://www.youtube.com/embed/gpPY-mqGXKs"
    title="Youtube video"
    width="100%"
  />
</div>
`;
